src/App.js

```diff
@@ -72,6 +79,8 @@ const styles = () => ({
   },
 });
 
+const queryClient = new QueryClient();
```

Comment:
can you pass the same default options as blsq report components (no magic refresh on focus gained, no retry)